packages/realm-web/scripts/postversion.ts

```diff
@@ -30,7 +30,7 @@ const changelogPath = path.resolve(__dirname, "../CHANGELOG.md");
 
 // Update the readme file to use the new version in the script-tag.
 const readmeContent = fs.readFileSync(readmePath, "utf8");
-const readmeContentReplaced = readmeContent.replace(/realm-web@[^/]+/, `realm-web@${version}`);
+const readmeContentReplaced = readmeContent.replace(/realm-web@[^"/]+/, `realm-web@${version}`);
```

Comment:
Is this change deliberate?

Reply:
Yeah. Now that the version is the last part of the URL it needed a " in the pattern.